lib/crowbar/client/command/upgrade/prepare.rb

```diff
@@ -34,7 +34,10 @@ def execute
             request.process do |request|
               case request.code
               when 200
-                say "Setting nodes to upgrade state"
+                say "Setting nodes to upgrade state." +
+                  " Please query the upgrade status and make sure that" +
+                  " 'upgrade_prepare' has passed before executing the next step."
```

Comment:
Is that warning necessary? Next step should not be able to execute when the previous one has failed (or is running)